### ISSUE TITLE
Hard to parse Golang time into ISO format

### ISSUE THREAD

**Issue by shaktals** [0chain/gosdk#170]
When a new wallet is generated, the peculiar golang `time` format is returned, i.e.:
`2021-08-03 15:07:12.563 -0300 UTC-3 m=+77.431000065`

It would be much easier for clients to handle that date if it followed a standard format like [IETF RFC3339](https://datatracker.ietf.org/doc/html/rfc3339), i.e.:
`2021-08-03T16:38:09-03:00`

I found 4 places where a `DateCreated` prop is set using `time.Now().String()`, i.e. [here](https://github.com/0chain/gosdk/blob/master/core/zcncrypto/bls0chain.go#L89).
The change is simple: `time.Now().Format(time.RFC3339)`.

I have made a branch for it, but I don't have push rights to the repo yet.
But if this is approved, I'll ask Saswata for it, push and PR.

ping @stewartie4 @Sriep

**Comment by stewartie4** [comment]
Nice catch, I agree we should be using standards where we can especially when it comes to the language agnostic API
